src/locales/zh-cn.js

```diff
@@ -0,0 +1,32 @@
+// numeral.js locale configuration
+// locale : Chinese for People's Republic of China (zh-cn)
+// author : Lee Hanxue : https://github.com/hanxue
+
+(function (global, factory) {
+    if (typeof define === 'function' && define.amd) {
+        define(['../numeral'], factory);
+    } else if (typeof module === 'object' && module.exports) {
+        factory(require('../numeral'));
+    } else {
+        factory(global.numeral);
+    }
+}(this, function (numeral) {
+    numeral.register('locale', 'zh-cn', {
```

Comment:
How is this different from chs locale?

Reply:
Arguably, the `chs` locale should be `zh-cn` as the other Numeral locales seem to use their ISO 639 locale codes but `chs` does not.